app/src/main/res/values/strings.xml

```diff
@@ -5,46 +5,49 @@
     <string name="comments_label">Comments</string>
     <string name="blogposts_label">Blog posts</string>
 
-    <!-- text for error message displayed in blogpost feature -->
-    <string name="error_message_title">Error</string>
-    <string name="error_message_message">An error occurred.</string>
-    <string name="error_message_dismiss_button">Okay</string>
-
     <!-- text for "no comments" comment to be displayed when there are no comments -->
     <string name="no_comments_text">No comments to display.</string>
 
+    <!-- AlertDialogFactory text -->
+    <string name="success_title">Success</string>
+    <string name="success_button">Okay</string>
+
+    <string name="error_title">Error</string>
+    <string name="error_button">Dismiss</string>
+
+    <!-- for MainActivity's dialog -->
+    <string name="reload_button">Reload</string>
+    <string name="exit_button">Exit</string>
+
+    <!-- Custom success and error messages -->
+
+    <string name="comment_form_success_message">Your comment is now pending approval.</string>
```

Comment:
Rather than store all of our resources in a single strings.xml file, is it possible to have smaller, more logically related xml files? The names could then be less redundant and it wouldnt have that Constants.java feel to it